skimage/restoration/tests/test_denoise.py

```diff
@@ -153,29 +153,29 @@ def test_denoise_tv_bregman_3d():
 
 
 def test_denoise_bilateral_2d():
-    img = checkerboard_gray.copy()
+    img = checkerboard_gray.copy()[:50,:50]
     # add some random noise
     img += 0.5 * img.std() * np.random.rand(*img.shape)
     img = np.clip(img, 0, 1)
 
     out1 = restoration.denoise_bilateral(img, sigma_color=0.1,
-                                         sigma_spatial=20, multichannel=False)
```

Comment:
Agreed.  I tried with 5 and 10, that didn't pass one of the tests. 

Re the window size, I assume the filters pad somehow in a way that makes sense.